po/de.po

```diff
@@ -1,26 +0,0 @@
-# starter-kit German translations
-#, fuzzy
-msgid ""
-msgstr ""
-"Project-Id-Version: starter-kit 1.0\n"
-"Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2018-06-19 17:07+0200\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
-"MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: 8bit\n"
-
-#: src/index.html:20
-msgid "Cockpit Starter Kit"
-msgstr "Cockpit Bausatz"
```

Comment:
Haha, I'm glad  it cleaned it up, as that's still the template from starter-kit :grinning: